docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/30-common-configurations.md

```diff
@@ -0,0 +1,423 @@
+[Secure Configuration](../40-security/10-secure-configuration.md) section for
+more information on securing the API server.
+:::
+
+To configure the API server to use an `Ingress` resource, set the following
+configuration:
+
+```yaml
+api:
+  ingress:
+    enabled: true
+    ingressClassName: nginx
+```
+
+#### Ingress TLS
+
+By default, Kargo will enable TLS on the `Ingress` resource using a self-signed
+certificate using [cert-manager](https://cert-manager.io). 
+
+:::note
+When making use of the self-signed certificate option, cert-manager must be
+installed in the cluster.
+:::
+
+To supply your own certificate, set the following configuration:
+
+```yaml
+api:
+  ingress:
+    tls:
+      selfSignedCert: false
+```
+
+When setting `selfSignedCert` to `false`, the `Ingress` resource expects a
+[TLS `Secret`](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)
+with the name `kargo-api-ingress-cert` to exist in the same namespace as the
+API server.
+
+## Git Configuration
+
+Kargo supports a number of Git-related configurations that can be set at
+installation time.
+
+### Default Commit Author
+
+To set the default commit author Kargo will use when committing changes using
+the [`git-commit` Promotion step](../../50-user-guide/60-reference-docs/30-promotion-steps/git-commit.md),
+the following configuration can be set (shown here with the default values):
+
+```yaml
+controller:
+  gitClient:
+    name: Kargo
+    email: no-reply@kargo.io
+```
+
+### Signing Commits
+
+To sign commits made by Kargo, a reference to a `Secret` (in the same namespace
+as Kargo is installed to) containing a signing key can be configured. The key
+should be a GPG key in ASCII-armored format  without a passphrase, under the
+key `signingKey`.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kargo-git-signing-key
+type: Opaque
+data:
+  signingKey: <base64-encoded-ascii-armored-gpg-key>
+```
+
+The `Secret` can then be referenced in the Kargo configuration.
+
+```yaml
+controller:
+  gitClient:
+    signingKeySecret:
+      name: kargo-git-signing-key
+      type: gpg
+```
+
+:::note
+When using a signing key, the `gitClient.name` and `gitClient.email`
+configuration options must match the name and email associated with the GPG
+key.
+:::
+
+## Argo CD Configuration
+
+Kargo supports a number of Argo CD-related configurations that can be set at
+installation time.
+
+### Disabling the Argo CD Integration
+
+By default, Kargo will enable the Argo CD integration, which configures Kargo
+to work with `Application` resources created by Argo CD. This can be disabled
+as follows:
+
+```yaml
+controller:
+  argocd:
+    integrationEnabled: false
+```
+
+When not enabled, the controller will not watch Argo CD `Application` resources
+and disable Argo CD specific features. Explicitly disabling is preferred if this
+integration is not desired, as it will grant fewer permissions to the controller.
```

Comment:
So we want to mention in this paragraph the behavior of degrading to a disabled state if the controller learns at startup that Argo CD CRDs are not present in the cluster?

I think that segues well into the point you're already making about explicitly disabling being preferable if the capability is not wanted.